clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -581,8 +594,29 @@
 
+    public static final Schema DESCRIBE_GROUPS_RESPONSE_GROUP_METADATA_V1 = new Schema(new Field("error_code", INT16),
+                                                                                       new Field("group_id",
+                                                                                                 STRING),
+                                                                                       new Field("state",
+                                                                                                 STRING,
+                                                                                                 "The current state of the group (one of: Dead, Stable, AwaitingSync, or PreparingRebalance, or empty if there is no active group)"),
+                                                                                       new Field("protocol_type",
+                                                                                                 STRING,
+                                                                                                 "The current group protocol type (will be empty if there is no active group)"),
+                                                                                       new Field("protocol",
+                                                                                                 STRING,
+                                                                                                 "The current group protocol (only provided if the group is Stable)"),
+                                                                                       new Field("members",
+                                                                                                 new ArrayOf(DESCRIBE_GROUPS_RESPONSE_MEMBER_V0),
+                                                                                                 "Current group members (only provided if the group is not Dead)"),
+                                                                                       new Field("offsets",
+                                                                                                 new ArrayOf(DESCRIBE_GROUPS_RESPONSE_OFFSET_V1),
+                                                                                                 "Offset records associated with this group"));
+
+    public static final Schema DESCRIBE_GROUPS_RESPONSE_V1 = new Schema(new Field("groups", new ArrayOf(DESCRIBE_GROUPS_RESPONSE_GROUP_METADATA_V1)));
+
     public static final Schema[] DESCRIBE_GROUPS_REQUEST = new Schema[] {DESCRIBE_GROUPS_REQUEST_V0};
-    public static final Schema[] DESCRIBE_GROUPS_RESPONSE = new Schema[] {DESCRIBE_GROUPS_RESPONSE_V0};
+    public static final Schema[] DESCRIBE_GROUPS_RESPONSE = new Schema[] {DESCRIBE_GROUPS_RESPONSE_V1};
```

Comment:
Thanks for bringing this up. I wasn't totally sure what to process is for changing the protocol, and whether I'm actually correct to assume that the protocol has to be changed for this JIRA. I haven't opened a KIP yet, if you think that's eventually going to be needed for I'd be happy to create one. Thanks in advance for clarifying this.